pkg/enactment/enactment.go

```diff
@@ -10,7 +10,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func CountByPolicy(cli client.Client, policy *nmstatev1beta1.NodeNetworkConfigurationPolicy) (int, enactmentconditions.ConditionCount, error) {
+func CountByPolicy(cli client.Reader, policy *nmstatev1beta1.NodeNetworkConfigurationPolicy) (int, enactmentconditions.ConditionCount, error) {
```

Comment:
needs to be updated to client.APIclient

Reply:
We expect this function to only read so we pass a reader.

Reply:
OK. understand. Thanks